spread.yaml

```diff
@@ -522,4 +522,44 @@ suites:
             distro_purge_package qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/performance/:
+        summary: Performance tests for snapd
+        manual: true
+        environment:
+            RES_FILE: "$PROJECT_PATH/test.perf"
+        prepare: |
+            . "$TESTSLIB/prepare.sh"
```

Comment:
You should use `"$TESTSLIB/prepare-restore.sh" --prepare-suite`, also `--prepare-suite-each`, `--restore-suite-each`, `--restore-suite`.